webview/src/setup/components/App.test.tsx

```diff
@@ -465,14 +470,29 @@ describe('App', () => {
         pythonBinPath: 'python',
         sectionCollapsed: undefined,
         shareLiveToStudio: false
-      })
-      mockPostMessage.mockClear()
-      const button = screen.getAllByText('Show Experiments')[0]
-      fireEvent.click(button)
-      expect(mockPostMessage).toHaveBeenCalledTimes(1)
-      expect(mockPostMessage).toHaveBeenCalledWith({
-        type: MessageFromWebviewType.OPEN_EXPERIMENTS_WEBVIEW
-      })
+      }
+
+      renderApp(dvcNotSetup)
+
+      const dvcDetails = within(
+        screen.getAllByTestId('section-container')[0]
```

Comment:
Oh wait, we probably shouldn't use array targeting as section order can change when we add more sections. Will update that.